gnome-extensions/install-extensions.sh

```
@@ -266,6 +323,74 @@ export_current_setup() {
+    # Critical extensions that must be enabled
+    local critical_extensions=(
+        "monitor@astraext.github.io"
+        "search-light@icedman.github.com"
+        "bluetooth-battery@michalw.github.com"
+        "bluetooth-quick-connect@bjarosze.gmail.com"
+        "tweaks-system-menu@extensions.gnome-shell.fifi.org"
+        "BringOutSubmenuOfPowerOffLogoutButton@pratap.fastmail.fm"
+        "PrivacyMenu@stuarthayhurst"
+    )
+    
+    local enabled_list=$(gnome-extensions list --enabled)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)